Jenkinsfile.content

```diff
@@ -9,14 +26,24 @@ node('vetsgov-general-purpose') {
   dockerContainer = commonStages.setup()
 
   stage("Build") {
-    commonStages.build(params.ref, dockerContainer, params.ref, params.env, false)
+    commonStages.build(ref, dockerContainer, ref, params.env, false)
   }
 
   stage("Prearchive") {
     commonStages.prearchive(dockerContainer, params.env)
   }
 
   stage("Archive") {
-    commonStages.archive(dockerContainer, params.ref, params.env)
+    commonStages.archive(dockerContainer, ref, params.env)
+  }
+
+  stage("Deploy Dev or Staging") {
+    if (params.env != "vagovprod" && params.deploy) {
+      commonStages.runDeploy("deploys/vets-website-${params.env}", ref, true)
+
+      slackSend(message:"Web content refresh in ${params.env} completed.",
+                color: "good",
+                channel: "#cms-notifications")
```

Comment:
The example in the `vagov-content` jenkins file of using the channel param doesn't have `#` in the channel name.